ansible_collections/arista/avd/roles/eos_cli_config_gen/README_v4.0.md

```diff
@@ -3288,7 +3291,7 @@ traffic_policies:
             log: < true | false (default false) >
           # Last resort policy
           default_actions:
-            < ipv4 | ipv6 >:
+            - ip: < ipv4 | ipv6 >
```

Comment:
I don't think `ip` is the right name here. I see three options:
1. Move back to a dict with two hardcoded values, since you can only have one of each in EOS. This would match the data model for `field_sets` above.
2. Use `type`. This would match the `policies` above
3. Use `address_family`. This would technically be most correct.

My preference is 1.
@tgodaA @carlbuchmann

Reply:
I will go with **1**, because this doesn't look like a list to me with 2 possible options though we can support it as a list.

Reply:
@Shivani-chourasiya please change to a dictionary:
```yaml
default_actions:
  ipv4:
    ...
  ipv6:
    ...
```

Reply:
okay

Reply:
Updated!